.brazil.json

```diff
@@ -1,8 +1,8 @@
 {
   "dependencies": {
     "org.jetbrains.kotlin:kotlin-stdlib:2.0.*": "KotlinStdlib-2.x",
-    "org.jetbrains.kotlinx:kotlinx-coroutines-core-jvm:1.8.*": "KotlinxCoroutinesCoreJvm-1.8.x",
-    "org.jetbrains.kotlinx:kotlinx-coroutines-jdk8:1.8.*": "KotlinxCoroutinesJdk8-1.8.x",
+    "org.jetbrains.kotlinx:kotlinx-coroutines-core-jvm:1.*": "KotlinxCoroutinesCoreJvm-1.x",
+    "org.jetbrains.kotlinx:kotlinx-coroutines-jdk8:1.*": "KotlinxCoroutinesJdk8-1.x",
```

Comment:
Question: does this mean we'll automatically update to the latest version, e.g. `1.10.x`?

Reply:
If it's imported internally, yes. Externally we would still remain on whatever version we defined in `gradle/libs.versions.toml`